packages/local_auth/local_auth_darwin/example/pubspec.yaml

```diff
@@ -9,7 +9,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  local_auth_ios:
+  local_auth_darwin:
     # When depending on this package from a real application you should use:
     #   local_auth: ^x.y.z
```

Comment:
This should be `local_auth_darwin:` (it was wrong before too, we just didn't catch it).